doc/04-Housekeeping.md

```diff
@@ -0,0 +1,19 @@
+# Icinga DB Housekeeping
```

Comment:
Should not have the "Icinga DB" prefix. Also, maybe something like "history cleanup" or "history retention" would be more descriptive.